src/coreclr/CMakeLists.txt

```diff
@@ -58,7 +58,7 @@ endif(NOT CLR_CROSS_COMPONENTS_BUILD AND CLR_CMAKE_BUILD_SUBSET_RUNTIME)
 #-------------------------------
 # Use EventPipe C library
 #-------------------------------
-set(FEATURE_PERFTRACING_C_LIB 1)
+#set(FEATURE_PERFTRACING_C_LIB 1)
```

Comment:
nit: maybe better place for `set(FEATURE_PERFTRACING_C_LIB 0)` is in this block: https://github.com/dotnet/runtime/blob/42a7b314d2bf786d9c25018e31317d3fa2c33203/src/coreclr/clrfeatures.cmake#L9-L11

Reply:
Had it there to begin with, but when properties was needed in debug_pal project that location was to late, https://github.com/dotnet/runtime/blob/f9bb1673708ca840da0e71f9a9444ea9b0d31911/src/coreclr/CMakeLists.txt#L87 since include happened after doing `add_subdirectory(src/debug/debug-pal)`

Reply:
But since this will be temporary until we switch full over to C library, it will probably just be run on some outerloop CI lane, so maybe it can be controlled by passing a cmake argument as part of that outloop CI lane. Then we can drop setting it from cmake files. Right now its mainly in here for running the different configs on CI for this PR.

Reply:
Ah, ok. could be by design that we want debug-pal to build that code in certain way, regardless of the state of feature define. If not, then you have spotted a bug. 😅
>  so maybe it can be controlled by passing a cmake argument as part of that outloop CI lane.

Yup, it's possible with `./build.sh clr --cmakeargs -DFEATURE_PERFTRACING_C_LIB=1` similar to:
https://github.com/dotnet/runtime/blob/ffc0b7aa63799732b8c863228cf99c5ec3301222/eng/pipelines/coreclr/templates/build-job.yml#L113